instances/events-committee.near/widget/devhub/entity/proposal/Proposal.jsx

```diff
@@ -345,12 +345,14 @@ const proposalStatusOptions = [
       status: TIMELINE_STATUS.REVIEW,
       sponsor_requested_review: false,
       reviewer_completed_attestation: false,
+      kyc_verified: false,
```

Comment:
before merging this, we should also deploy the latest changes to events contract, else it might throw error.

Reply:
Yes for sure. Update the indexer as well, see this issue: https://github.com/NEAR-DevHub/neardevhub-contract/issues/144